javascript/javascript1/week2/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
@27ManishaPatel  you code is not working on the console kindly check 
Failed to load resource: net::ERR_FILE_NOT_FOUND